cle/backends/relocations/peamd64.py

```diff
@@ -0,0 +1,21 @@
+import logging
+from . import generic_pe
+
+l = logging.getLogger('cle.relocations.peamd64')
+
+arch = 'peAMD64'
```

Comment:
Similarly, `Relocation` has some ELF-specific things in it, like `is_rela` and `addend`. With separate subdirs within the backends, these things could be pieced out to an `ELFReloc`.